setup.py

```diff
@@ -22,13 +22,9 @@
     package_dir={"": "src"},
     py_modules=["pybreaker"],
     include_package_data=True,
+    install_requires=["typing-extensions>=3.10.0; python < '3.8'"],
```

Comment:
[`Literal`](https://docs.python.org/3/library/typing.html#typing.Literal) was introduced in Python 3.8. `typing_extensions` is the official backport package for typing classes. 

Python 3.7 is almost at EOL (December 2022).